src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -189,8 +192,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         this.updateContextOfActiveNotebookKernel(activeEditor);
     }
     private onDidChangeActiveTextEditor(e?: TextEditor) {
-        this.isPythonFileActive =
-            e?.document.languageId === PYTHON_LANGUAGE && !this.notebookEditorProvider.activeEditor;
```

Comment:
This is another spot that I believe was incorrect. ```this.notebookEditorProvider.activeEditor``` is always available as long as a notebook was opened.

Reply:
But this might affect other things that I don't realize. This is why I think we shouldn't port this change.

Reply:
I'd have to look a bit deeper myself, but then would the actual fix here be to make notebookEditorProvider.activeEditor undefined if it doesn't actually have editor focus?

Reply:
Not sure I'd  want to do that in case there was things depending upon it but yes we should probably change that property to use the window.activeNotebookEditor instead.

Reply:
well come to think of it, if I enter an issue for it, we'll just never fix it. So I'll just change it now.